mm2src/coins/utxo/utxo_common.rs

```diff
@@ -3114,11 +3119,11 @@ pub fn validate_payment<T: UtxoCommonOps>(
                 Ok(t) => t,
                 Err(e) => {
                     if attempts > 2 {
-                        return ERR!(
+                        return MmError::err(ValidatePaymentError::InternalError(format!(
                             "Got error {:?} after 3 attempts of getting tx {:?} from RPC",
                             e,
-                            tx.tx_hash()
-                        );
+                            tx.tx_hash(),
+                        )));
```

Comment:
```
return MmError::err(ValidatePaymentError::UtxoRpcError(UtxoRpcError::Internal(format!(
                            "Got error {:?} after 3 attempts of getting tx {:?} from RPC",
                            e,
                            tx.tx_hash(),
                        ))));
```
This might look a bit much but this error is not internal but a `UtxoRpcError`